components/elm/src/data_types/ColumnDataType.F90

```diff
@@ -5830,26 +5843,26 @@ subroutine col_wf_init(this, begc, endc)
           avgflag='A', long_name='column-integrated snow freezing rate', &
            ptr_col=this%qflx_snofrz, set_lake=spval, c2l_scale_type='urbanf', default='inactive')
 
-    if (create_glacier_mec_landunit) then
+    !if (create_glacier_mec_landunit) then
        this%qflx_glcice(begc:endc) = spval
         call hist_addfld1d (fname='QICE',  units='mm/s',  &
              avgflag='A', long_name='ice growth/melt', &
               ptr_col=this%qflx_glcice, l2g_scale_type='ice')
-    end if
+    !end if
 
-    if (create_glacier_mec_landunit) then
+    !if (create_glacier_mec_landunit) then
        this%qflx_glcice_frz(begc:endc) = spval
         call hist_addfld1d (fname='QICE_FRZ',  units='mm/s',  &
              avgflag='A', long_name='ice growth', &
               ptr_col=this%qflx_glcice_frz, l2g_scale_type='ice')
-    end if
+    !end if
 
-    if (create_glacier_mec_landunit) then
+    !if (create_glacier_mec_landunit) then
        this%qflx_glcice_melt(begc:endc) = spval
         call hist_addfld1d (fname='QICE_MELT',  units='mm/s',  &
              avgflag='A', long_name='ice melt', &
               ptr_col=this%qflx_glcice_melt, l2g_scale_type='ice')
-    end if
+    !end if
```

Comment:
Please delete the commented out `if` and `endif` lines, and fix the indenting of the remaining lines.